config/jobs/kubernetes-sigs/controller-runtime/controller-runtime-presubmits-release-0.12.yaml

```
@@ -0,0 +1,34 @@
+presubmits:
+  kubernetes-sigs/controller-runtime:
+  - name: pull-controller-runtime-test-release-0-12
+    decorate: true
+    always_run: true
+    path_alias: sigs.k8s.io/controller-runtime
+    branches:
+    - ^release-0.12$
+    spec:
+      containers:
+      - image: golang:1.17
```

```suggestion
      - image: golang:1.18
```

I am unsure about this, but shouldn't we build the release branch with Go 1.18?

---

Oh good catch! +1 

/hold
for bump to Go 1.18

---

Terrific catch!

---

done